_data/challenges.yml

```diff
@@ -150,7 +152,7 @@
   prizes:
     - name: Powerbank
       num: 1
-      link:
+      link: http://i.imgur.com/QRGu9kT.jpg
```

Comment:
@xicon73 If you can link the Raspberry Pi 3 official link here so people can find out more :)